### PR TITLE
MSI: #206 - Added strict type static code test

### PR DESCRIPTION
**Pull request by DwayneH**
Added static analysis strict_type test to the LiveCodeTest as discussed in #206.

### CODE REVIEW

File: dev/tests/static/testsuite/Magento/Test/Php/LiveCodeTest.php [hunk 1]
```diff
@@ -266,4 +271,22 @@ public function testCopyPaste()
             "PHP Copy/Paste Detector has found error(s):" . PHP_EOL . $output
         );
     }
+
```
Comment (maghamed) [inline dev/tests/static/testsuite/Magento/Test/Php/LiveCodeTest.php:274]: Please provide a description for this method in Comment block before the method

**Review by maghamed — CHANGES_REQUESTED** [review]
Please also provide a blacklist for files which we should exclude from this validation.

For example, we are using legacy code which relies on Duck Typing, and refactoring to comply with Strict Types would cost us a lot. 
So, we would just add this file to the Blacklist (files which we skip)

**Comment by maghamed** [comment]
Hey @DwayneH  could you please address comments I mentioned above

**Comment by DwayneH** [comment]
I'll work on it! 👍

**Comment by DwayneH** [comment]
@maghamed Blacklisting and description are now included

**Review by maghamed — APPROVED** [review]

**Comment by maghamed** [comment]
Thanks, @DwayneH  for your first #MSI contribution! 
Good start! Keep it rolling!

**Pull request merged**